tests/nfs/test_nfs_multiple_operations_for_upgrade.py

```diff
@@ -7,12 +7,31 @@
 log = Log(__name__)
 
 
+def lookup_in_directory(client, mount):
+    """
+    Check if the mount point is accessible and contains expected files.
+    Args:
+        client: Client node where the mount is performed.
+        mount: Mount point path.
+    Returns:
+        bool: True if the mount point is accessible and contains expected files, False otherwise.
+    """
+    try:
+        out, _ = client.exec_command(sudo=True, cmd=f"ls {mount}")
```

Comment:
can't we use method - https://github.com/red-hat-storage/cephci/blob/main/ceph/ceph.py#L2229